docs/website/docs/development/architecture/podman-limitations.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Podman limitations
```

Comment:
This file should be part of the PR for https://github.com/redhat-developer/odo/issues/6592, no? To me, the content seems to not be complete at this point..

Reply:
Oups, yes, I'm removing it

Reply:
Thanks.

/lgtm